scripts/ccpp_prebuild.py

```diff
@@ -160,6 +161,19 @@ def clean_files(config):
     execute(cmd)
     return success
 
+def get_all_suites(suites_dir):
+    success = False
+    logging.info("No suites were given, compiling a list of all suites")
+    sdfs = []
+    for f in os.listdir(suites_dir):
+        match = SUITE_DEFINITION_FILENAME_PATTERN.match(f)
+        if match:
+            logging.info('Adding suite definition file {}'.format(f))
+            sdfs.append(f)
+    if sdfs:
+        success = True
+    return (success, sdfs)
+
```

Comment:
This seems like a lot of work to do instead of something like:
```
sdfs = glob.glob(suite_dir, "suite_*.xml")
```

Reply:
That's right ... when I was making those changes I wasn't sure whether I needed to get the suite name (which comes for free as `match.group(1)` with my approach) or not. Turned out I didn't.

In any case, even the more expensive approach costs probably less than a tenth of a second, so that should be ok.